Gemfile

```diff
@@ -36,7 +36,7 @@ gem 'bootsnap', '>= 1.1.0', require: false
 
 group :development, :test do
   gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
-  gem 'capybara', '~> 2.13.0'
+  gem 'capybara'
```

Comment:
Is it worth specifying that there's a requirement for `~3.3` here?

Reply:
Good catch. Let's use `~> 3.0` as specified [here](https://github.com/rails/rails/pull/30959#discussion_r181931451).